src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -61,29 +62,18 @@ export class OverflowMenuOption {
 
 	public tabIndex = -1;
 
+	public title = null;
```

Comment:
Maybe this can be a getter so it updates automatically like it used to?

Reply:
Nope, that was also causing some change after check errors ... the component instance shouldn't be reused in this case anyway, and if it is I'd expect `AfterViewInit` to be called again

Reply:
Cool. Do you mind adding that explanation as a comment?

Reply:
Yep 👍